storm-client/src/jvm/org/apache/storm/grouping/LoadAwareCustomStreamGrouping.java

```diff
@@ -20,5 +20,6 @@
 import java.util.List;
 
 public interface LoadAwareCustomStreamGrouping extends CustomStreamGrouping {
+   void refreshLoad(LoadMapping loadMapping);
    List<Integer> chooseTasks(int taskId, List<Object> values, LoadMapping load);
```

Comment:
If we are going to refresh the load out of band then lets delete the LoadMapping from the parameters passed into chooseTask.